### ISSUE TITLE
Show suggestions even when trigger is preceded with non whitespace.

### ISSUE THREAD

**Issue by prashantagarwal** [draft-js-plugins/draft-js-plugins#577]
There are valid use cases when it is required that suggestions show up even when trigger is preceded with non whitespace.

For example:
Lets say in case of url. 
`www.abc.com/@prashantagrawal`

**Comment by Revln9** [comment]
the default regex used for triggering the suggestion list has some assumptions , at `mentionSuggestionsStrategy.js` you have this line

`  (0, _findWithRegex2.default)(new RegExp('(\\s|^)' + (0, _lodash2.default)(trigger) + regExp, 'g'), contentBlock, callback);`

so this regex  `(\\s|^)` is used to get the left mention bound, this regex match either a whitespace or a the beginning of the line 

so basically speaking you need to get rid of it  ! BUT this is not the only thing that is used for delimiting the text to replace , there is another file called getWordAt.js at the utils folder that you need (or not , not tested yet ) to tweak. 

Lastly here is what i came up with for the regex that you need , http://regexr.com/3epp2

Hope it helps !

**Comment by tommoor** [comment]
@prashantagarwal why did you create this duplicate issue when you'd already replied in https://github.com/draft-js-plugins/draft-js-plugins/issues/544 ?

**Comment by prashantagarwal** [comment]
@tommoor yeah, I realised later. Sorry, my bad.

**Comment by tommoor** [comment]
Cool, you should close this dupe one to help the maintainers out 💪

**Comment by prashantagarwal** [comment]
@Revln9 still no luck.
So let me explain you want I wanted to achieve.

Basically I want user to enter tokens using mentions in url.
eg.

https://abc.com/{{token1}}/{{token2}}/finally

here:
'{' is the mention trigger.
'{{' is the mentionPrefix.
'}}' is the mentionSuffix (support for which I added in my fork)

also I removed the trailing whitespace that was being entered after every mention is inserted.

The only thing left here to do is show popup whenever the mention Trigger is entered and it should not be preceded with whitespace.
